e2e/cypress/integration/account_settings/display/clock_display_mode_spec.js

```diff
@@ -76,6 +77,24 @@ describe('Account Settings > Display > Clock Display Mode', () => {
             verifyClockFormatIs24HourForPostWithMessage(postId, replyMessage2);
         });
     });
+
+    it('MM-T2096_2 Clock Display - Can switch from 12-hr to 24-hr', () => {
+        cy.apiAdminLogin().then(({user}) => {
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+            // # Set clock display to 24-hour
+            setClockDisplayTo24Hour();
+
+            // # Post a message with time after 1pm
+            const now = new Date();
+            const nextYear = now.getFullYear() + 1;
+            const futureDate = Date.UTC(nextYear, 0, 5, 14, 37); // Jan 5, 2:37pm
+            cy.postMessageAs({sender: user, message: 'Hello from Jan 5, 2:37pm', channelId: testChannel.id, createAt: futureDate});
+
+            // * Message posted shows timestamp in 24-hour format, e.g. 14:37
+            cy.getLastPost().find('time').should('contain', '14:37').and('have.attr', 'datetime', `${nextYear}-01-05T14:37:00.000Z`);
```

Comment:
Hi @saturninoabril 
The test repeatedly fails at this step as shown in this screenshot. 
![Screenshot 2020-09-14 at 3 20 25 PM](https://user-images.githubusercontent.com/1429138/93071809-386aa400-f69e-11ea-8aed-02e4532c1e9e.png)

I believe the issue is because the ExperimentalTimezone is not enabled during the test. When this is enabled, the test starts working fine. Can you add the below steps in the before block before doing the apiInitSetup. 

```
        // # Update Configs
        cy.apiUpdateConfig({
            DisplaySettings: {
                ExperimentalTimezone: true,
            }
        });  
```

Reply:
Good catch @srkgupta! Now updated, PTAL again.  Thanks!